cms/djangoapps/contentstore/tests/tests.py

```diff
@@ -302,6 +309,47 @@ def test_inactive_session_timeout(self):
         # re-request, and we should get a redirect to login page
         self.assertRedirects(resp, settings.LOGIN_REDIRECT_URL + '?next=/home/')
 
+    @data(*MICROSITE_BACKENDS)
+    def test_signup_button_index_page(self, site_backend):
```

Comment:
Use [SiteMixin](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/site_configuration/tests/mixins.py#L7) instead of the microsite scaffolding to test the site configuration override.